### ISSUE TITLE
'aclocal-1.14' is missing on your system.

### ISSUE THREAD

**Issue by ghost** [rgamble/libcsv#25]
Current version of automake is 1.16.2, and the build will fail if version does not match. One way to fix this is to refer this [post](https://stackoverflow.com/questions/33278928/how-to-overcome-aclocal-1-15-is-missing-on-your-system-warning). But in order to use `autoreconf`, you may have to install [libtools](https://pkgs.org/download/libtool). Another problem that might occur during the build process is that `make install` has to be executed as root.

TLDR; Added commands before for simplification. Just run them.

```bash
autoreconf -f -i
./configure && make && make check && sudo make install
```

Although it is not added in the README, if you want to uninstall this library, you simply have to run:
```bash
sudo make uninstall
```

It would still be better if the owner of the repository patches with the new changes.
